### ISSUE TITLE
Language values not populated

### ISSUE THREAD

**Issue by drpeck** [bomortensen/Our.Umbraco.OpeningHours#8]
I installed via Nuget and the Umbraco package repo and I seem to be missing some language files. I'm guessing this is a bug?

![image](https://cloud.githubusercontent.com/assets/1196047/16683961/412893ea-44fa-11e6-9926-2afc26c284d9.png)

**Comment by drpeck** [comment]
I'm using 7.3.4 BTW, and I can see your language files in the App_Plugin folder.

**Comment by abjerner** [comment]
@drpeck AFAIK this is fixed in the latest code on GitHub, but I'll have a look at it just to be sure. What language are you using - `en`, `en-US` or something?

**Comment by drpeck** [comment]
The days of the week appear. Maybe it was an app recycle thing, but I had thought I did that. The only remaining issue is the holidayTitle placeholder is still there. The reason for that is that the language file has holiday**s**Title instead of the singular holidayTitle. I'm using en-GB but en-US has the same issue.

**Comment by drpeck** [comment]
Checking the source code I can see that you've already found this. I guess it just hasn't made it in to the build?

**Comment by JanCoyle** [comment]
Hej Bo

Jeg fik også denne første gang jeg lavede visningen, men du skal bare lave en opdatering/refresh så kommer den hos mig.

Med venlig hilsen / kind regards
Jan Borup Coyle
[gitLogo_Payoff]
Vandværksvej 24, 5000 Odense C
http://www.g-it.dk<http://www.g-it.dk/>
Mobil: 25406625
CVR: 37372862

Fra: David Peck [mailto:notifications@github.com]
Sendt: 8. juli 2016 11:54
Til: bomortensen/Our.Umbraco.OpeningHours <Our.Umbraco.OpeningHours@noreply.github.com>
Emne: [bomortensen/Our.Umbraco.OpeningHours] Language values not populated (#8)


I installed via Nuget and the Umbraco package repo and I seem to be missing some language files. I'm guessing this is a bug?

[image]<https://cloud.githubusercontent.com/assets/1196047/16683961/412893ea-44fa-11e6-9926-2afc26c284d9.png>

**Comment by abjerner** [comment]
@drpeck Thanks for reporting. I think most of the issues in the beta are fixed, but I simply haven't had the time to push a new release.

I took over this project from Bo because Bo didn't have the time, and I thought I did. Then I hit period with a lot of overwork, so I haven't really had the time to wrap up the code for a stable release yet. However the good thing is that I will now begin to have some more time, and I will try to give this project some well deserved focus.

I don't really like giving ETAs for spare time projects, but it is my goal to have v1 released by the end of this week. It is possible that something else comes up, but I will at least give it a try ;)
